algo/src/Sort/Sort.java

```diff
@@ -0,0 +1,10 @@
+package Sort;
+
+/**
+ * Created by abhilabh on 1/24/19.
+ */
+public interface Sort {
+    public void quickSort(int[] a,int start,int end);
```

Comment:
Please add only one method for example perform()

And then create multiple classes like Quicksort, Mergesort etc and implement perform method